test/test_special_elixirs.jl

```diff
@@ -132,13 +132,9 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples")
 
     J = jacobian_ad_forward(semi)
     @test Matrix(A) ≈ J
-    λ = eigvals(J)
-    @test maximum(real, λ) < 10 * sqrt(eps(real(semi)))
```

Comment:
Out of curiosity: why remove?

Reply:
We had a test of the eigenvalues of `A` above and we check `J ≈ A` here. Thus, it's just an additional computation that takes time without giving us more information.